composites/OnboardingWizard/OnboardingWizard.js

```diff
@@ -131,6 +134,11 @@ class OnboardingWizard extends React.Component {
 	 * @returns {Object}  The first step object
 	 */
 	getFirstStep( steps ) {
+		const firstStep = this.props.startingStep;
+		if ( firstStep !== "" ) {
+			return firstStep;
```

Comment:
This line isn't covered?

Reply:
I've created a technical debt issue: https://github.com/Yoast/yoast-components/issues/824